landing/templates/landing/index.html

```diff
@@ -1,8 +1,59 @@
-{% extends 'base.html' %}
+{% load staticfiles %}
 
-{% block title %}Welcome!{% endblock %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Social Distribution - Welcome</title>
 
-{% block content %}
-    <h1>Welcome to Social Distribution</h1>
-    <p>Please <a href="{% url 'login' %}">sign in</a> to continue!</p>
-{% endblock %}
\ No newline at end of file
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
+    <link href="{% static 'css/style.css' %}" rel="stylesheet">
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
```

Comment:
I assume that this just came in with some Bootstrap code, but we're in the enviable position of not needing to worry about support old IE, so we can delete this stuff!